docs/guides/run-elasticsearch-local.rst

```diff
@@ -0,0 +1,99 @@
+==========================================
+Enabling Elasticsearch on the local server
+==========================================
+
+Read the Docs has been using Elasticsearch which is a platform for distributed search and analysis of data in real time. To enable the search feature on your local installation, you need to install the elasticsearch locally and run the Elastic server. 
+
+Installation has been mainly divided into following steps.
+
+1. Installing Java
+------------------
+
+Elasticsearch requires JAVA 8 or later. Use `Oracle official documentation <http://www.oracle.com/technetwork/java/javase/downloads/index.html>`_. 
+or opensource distribution like `OpenJDK <http://openjdk.java.net/install/>`_.
+
+After installing java, verify the installation by,::
+
+	$ java -version
```

Comment:
Style nit: Please don't use tab characters.

Reply:
do you mean $ sign?

Reply:
Sorry, I wasn't clear in my earlier comment. I meant don't use the `\t` character to indent code. See the following comparison:

```rst
# With using tab:
	$ java -version

# Without tabs, used four spaces instead:
    $ java -version
```

Reply:
changed as mentioned :+1: